app/src/main/java/com/github/sdpsharelook/MainActivity.kt

```diff
@@ -1,24 +1,18 @@
 package com.github.sdpsharelook
 
```

Comment:
I don't really understand why the different calls to startActivity are commented, otherwise everything seems fine.